pkg/scheduler/objects/queue.go

```diff
@@ -1220,6 +1220,63 @@ func (sq *Queue) GetMaxResource() *resources.Resource {
 	return sq.internalGetMax(limit)
 }
 
+// Starting with the root, descend down to the target queue allowing children to override Resource values .
+// If the root includes an explicit 0 value for a Resource, do not include it in the accumulator and treat it as missing.
+// If no children provide a maximum capacity override, the resulting value will be the value found on the Root.
+// It is useful for fair-scheduling to allow a ratio to be produced representing the rough utilization % of a given queue.
+func (sq *Queue) GetFairMaxResource() *resources.Resource {
+	var limit *resources.Resource
+	if sq.parent == nil {
+		cleanedRootResources := resources.NewResource()
```

Comment:
Nit: no need to use long variable names in tiny scopes. Just call it "ret" or "out" or even "cleaned".

Reply:
done